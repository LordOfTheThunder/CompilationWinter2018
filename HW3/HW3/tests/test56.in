void TestIfBool() {
	int x;
	if (x) {
		int t;
	}
}
