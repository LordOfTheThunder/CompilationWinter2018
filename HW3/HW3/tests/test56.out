---end scope---
t INT 1
---end scope---
line 3: type mismatch
